__tests__/app.test.js

```diff
@@ -113,3 +113,52 @@ describe.only("GET /api/articles", () => {
       });
   });
 });
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns an array of comments objects", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: Returns an array of comments objects with comment_id in ASC order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          let prevCommentId = 0;
+          expect(comment.comment_id).toBeGreaterThan(prevCommentId);
+          prevCommentId = comment.comment_id;
+        });
+      });
+  });
+  test("404: Returns 'No comments found for this article'", () => {
+    return request(app)
+      .get("/api/articles/37/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No comments found for this article");
+      });
+  });
+  test("500: Returns 'Internal Server Error'", () => {
+    return request(app)
+      .get("/api/articles/number/comments")
```

Comment:
This instance should be a bad request not an internal server error. The error comes from the client passing an invalid article id